src/EditorFeatures/Core/Implementation/GoToDefinition/GoToDefinitionHelpers.cs

```diff
@@ -98,7 +99,18 @@ internal static class GoToDefinitionHelpers
         public static bool TryGoToDefinition(
             ImmutableArray<INavigableItem> navigableItems,
             string title,
-            Workspace workspace,
+            Workspace originalWorkspace,
```

Comment:
I see, but still this doesn't require anything from workspace except option, so why not just pass in option?